files/migrations.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/python3
+"""
+Runs migration scripts for this project
+
+Intended to be triggered by the %post rpm scriptlet rather than manually.
+"""
+
+import logging as log
```

Comment:
nit: I'd prefer if we just imported standard modules using their standard names, it confused me for a while because I thought `log` was a logger rather than the logging module itself.